src/shared/components/loader/ballTriangle/ballTriangle.tsx

```diff
@@ -0,0 +1,25 @@
+import { BallTriangle } from 'react-loader-spinner';
+
+interface IBallTriangleLoaderPropsModel {
+	load: boolean;
+	height: string;
+	width: string;
+	color: string;
```

Comment:
```suggestion
	load?: boolean;
	height: string;
	width: string;
	color?: string;
```
add Optional Chaining (?)

Reply:
Done